strings/valentines.json

```diff
@@ -99,6 +99,7 @@
 		"A heart-shaped candy that reads: WAG MY TAIL",
 		"A heart-shaped candy that reads: PDA",
 		"A heart-shaped candy that reads: DEVILS KISS",
-		"A heart-shaped candy that reads: CHEMISTRY"
+		"A heart-shaped candy that reads: CHEMISTRY",
+		"A heart-shaped candy that reads: LUV"
```

Comment:
God this copy paste sucks.

But it's also not your fault or problem, just fix the indentation in the file while you're here.

Reply:
Is it better now?  Sry, I'm new to github and byond coding.

Reply:
dw, they were commenting about how "A heart-shaped candy that reads:" is repeated every line. You didn't do anything wrong